### PR TITLE
Reconcile apiserver classic load balancer

### PR DESCRIPTION
**Pull request by vincepri**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by vincepri** [comment]
This PR is based off the work from @ashish-amarnath in #115

File: cloud/aws/providerconfig/v1alpha1/types.go [hunk 1]
```diff
@@ -214,6 +217,13 @@ func (s *Subnet) String() string {
 	return fmt.Sprintf("id=%s/az=%s/public=%v", s.ID, s.AvailabilityZone, s.IsPublic)
 }
 
+// ELB defines an AWS classic load balancer.
+type ELB struct {
+	ID string `json:"id"`
+
```
Comment (chuckha) [inline cloud/aws/providerconfig/v1alpha1/types.go]: any chance of putting DNS name in here?

File: cloud/aws/services/elb/loadbalancer.go [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+		return nil, NewNotFound(errors.Errorf("no classic load balancer found with name %q", name))
+	} else if err != nil {
+		return nil, errors.Wrapf(err, "failed to describe classic load balancer: %s", name)
+	}
+
+	return fromSDKTypeToClassicELB(out.LoadBalancerDescriptions[0]), nil
+}
+
+func fromSDKTypeToClassicELB(v *elb.LoadBalancerDescription) *v1alpha1.ClassicELB {
+	lb := &v1alpha1.ClassicELB{
```
Comment (chuckha) [inline cloud/aws/services/elb/loadbalancer.go]: is there any reason this isn't a direct return? if it's style preference please leave it, i'm just curious

**Pull request merged**